### PR TITLE
FEAT: ADDED PASSWORD MODIFICATION TO /ME ENDPOINT

### PR DESCRIPTION
**Pull request by Alex-GF**

### CODE REVIEW

File: server/tests/users.test.ts [hunk 1]
```diff
@@ -515,10 +514,10 @@ describe(`DELETE ${TEST_URLS.usersMe}`, () => {
         beforeAll(async () => {
             const response = await request(BASE_URL).post(TEST_URLS.newUser).send(TEST_USER)
             expect(response.statusCode).toBe(201)
-            token = response.body.data
+            token = response.body.data.token
         })
     
-        it('Should return 200', async () => {
+        it('Should return 201', async () => {
```
Comment (javiercavlop) [inline server/tests/users.test.ts]: If it is 201 why are you evaluating 200 in line 522?

File: server/db/populateInitial.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ function populateUsers() {
         firstName: 'Maria', 
         lastName: 'Doe',
         username: 'mariaDoe',
-        password: bcrypt.hashSync('maria1234', salt),
```
Comment (javiercavlop) [inline server/db/populateInitial.ts]: Both johnDoe and mariaDoe passwords were changed to have at least 8 characters

File: .gitignore [hunk 3]
```diff
@@ -158,5 +158,7 @@ GoogleCloudKey.json
 
 # Prod variables
 terraform/.env.prod
+.terraform/
```
Comment (javiercavlop) [inline .gitignore:161]: Should terraform folder be ignored?

**Review by javiercavlop — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (Alex-GF) [reply .gitignore:161]: It should, as terraform executes in GitHub Actions VM

File: server/db/populateInitial.ts (hunk 2 above)
Comment (Alex-GF) [reply server/db/populateInitial.ts]: I thought it was a mistake, I'll update postman and reset their values

File: server/tests/users.test.ts (hunk 1 above)
Comment (Alex-GF) [reply server/tests/users.test.ts]: It was a mistake while following the error, I'll reset the text

File: .gitignore (hunk 3 above)
Comment (javiercavlop) [reply .gitignore:161]: Ah yes. I did not see the . before all so I thought it was the whole terraform/ folder 😅

File: server/db/populateInitial.ts (hunk 2 above)
Comment (javiercavlop) [reply server/db/populateInitial.ts]: 👍🏻

File: server/tests/users.test.ts (hunk 1 above)
Comment (javiercavlop) [reply server/tests/users.test.ts]: 👌🏻

**Review by javiercavlop — APPROVED** [review]
Everything is fine. Well done!👏🏻

**Pull request merged**